### PR TITLE
Adding tests for rke templates with monitoring, cis scan enabled

### PR DESCRIPTION
**Pull request by soumyalj**

### CODE REVIEW

File: tests/validation/tests/v3_api/test_cluster_templates.py [hunk 1]
```diff
@@ -4,6 +4,10 @@
 from .test_rbac import create_user
 from rancher import ApiError
 import requests
+from .test_monitoring import validate_cluster_graph
+from .test_monitoring import cluster_query_template
+import copy
```
Comment (izaac) [inline tests/validation/tests/v3_api/test_cluster_templates.py:9]: Please group the imports as suggested by Python style:

- Standard library imports.
- Related third party imports.
- Local application/library specific imports.

And each import group sorted alphabetically.

In this case:

```python
import copy
import os
import pytest
import requests
from rancher import ApiError
from .common import *  # NOQA
from .test_monitoring import cluster_query_template
from .test_monitoring import validate_cluster_graph
from .test_rbac import create_user
```

File: tests/validation/tests/v3_api/test_cluster_templates.py [hunk 2]
```diff
@@ -609,15 +758,15 @@ def cluster_template_create_edit(userToken):
...
     client = userclient
     do_cloud_credential_config = {"accessToken": DO_ACCESSKEY}
     do_cloud_credential = client.create_cloud_credential(
         digitaloceancredentialConfig=do_cloud_credential_config)
     time.sleep(3)
     node_template = client.create_node_template(
         digitaloceanConfig={"region": "nyc3",
-                            "size": "4gb",
+                            "size": nodesize,
                             "image": "ubuntu-16-04-x64"},
```
Comment (izaac) [inline tests/validation/tests/v3_api/test_cluster_templates.py:770]: Can we use Ubuntu 18.03?

File: tests/validation/tests/v3_api/test_cluster_templates.py [hunk 3]
```diff
@@ -609,15 +758,15 @@ def cluster_template_create_edit(userToken):
     cluster_cleanup(client, cluster)
 
 
-def node_template_do(userclient):
+def node_template_do(userclient, nodesize):
```
Comment (izaac) [inline tests/validation/tests/v3_api/test_cluster_templates.py:761]: Question here, there's an existing `node_template_do` fixture in `test_rke_cluster_provisioning`. Are we sure these names won't collide at run time if we execute `test_rke_cluster_provisioning` and `test_cluster_templates` with pytest?

File: tests/validation/tests/v3_api/test_cluster_templates.py (hunk 1 above)
Comment (soumyalj) [reply tests/validation/tests/v3_api/test_cluster_templates.py:9]: Done

File: tests/validation/tests/v3_api/test_cluster_templates.py (hunk 2 above)
Comment (soumyalj) [reply tests/validation/tests/v3_api/test_cluster_templates.py]: Updated to Ubuntu 18.04. Also, updated ubuntu version and engineinstall url in rke_cluster_provisioning.py

File: tests/validation/tests/v3_api/test_cluster_templates.py (hunk 3 above)
Comment (soumyalj) [reply tests/validation/tests/v3_api/test_cluster_templates.py]: Should not have any effect. Renamed the method ...

File: tests/validation/tests/v3_api/test_cluster_templates.py [hunk 4]
```diff
@@ -1,9 +1,14 @@
+import copy
 import os
 import pytest
+import requests
+from rancher import ApiError
 from .common import *  # NOQA
+from .test_monitoring import cluster_query_template
+from .test_rke_cluster_provisioning import  engine_install_url
```
Comment (izaac) [inline tests/validation/tests/v3_api/test_cluster_templates.py:8]: Order this group alphabetically.

File: tests/validation/tests/v3_api/test_cluster_templates.py [hunk 5]
```diff
@@ -70,6 +97,26 @@ def get_cluster_config(k8sversion):
             "enabled": "true",
             "type": "localClusterAuthEndpoint"
         },
+        "scheduledClusterScan": {
+            "enabled": "true",
+            "scanConfig": {
+                "cisScanConfig": {
+                    "debugMaster": "false",
+                    "debugWorker": "false",
+                    "overrideBenchmarkVersion": "rke-cis-1.4",
```
Comment (izaac) [inline tests/validation/tests/v3_api/test_cluster_templates.py:106]: Should we make use of `from .common import CIS_SCAN_PROFILE` here ?

File: tests/validation/tests/v3_api/test_cluster_templates.py [hunk 6]
```diff
@@ -609,26 +761,28 @@ def cluster_template_create_edit(userToken):
...
         digitaloceanConfig={"region": "nyc3",
-                            "size": "4gb",
-                            "image": "ubuntu-16-04-x64"},
+                            "size": nodesize,
+                            "image": "ubuntu-18-04-x64"},
         name=random_name(),
         driver="digitalocean",
         namespaceId="dig",
         cloudCredentialId=do_cloud_credential.id,
+        engineInstallURL=engine_install_url,
```
Comment (izaac) [inline tests/validation/tests/v3_api/test_cluster_templates.py:778]: Good catch.

File: tests/validation/tests/v3_api/test_cluster_templates.py (hunk 5 above)
Comment (soumyalj) [reply tests/validation/tests/v3_api/test_cluster_templates.py]: Done

File: tests/validation/tests/v3_api/test_cluster_templates.py (hunk 4 above)
Comment (soumyalj) [reply tests/validation/tests/v3_api/test_cluster_templates.py]: Done

**Pull request merged**